extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -180,29 +170,21 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 						},
 						async (_progress, _token): Promise<void> => {
 							try {
+								let cr = this.schedulingParamsToEdit(this.saveArgs.coresRequest.w, this.saveArgs.coresRequest.c);
+								let cl = this.schedulingParamsToEdit(this.saveArgs.coresLimit.w, this.saveArgs.coresLimit.c);
+								let mr = this.schedulingParamsToEdit(this.saveArgs.memoryRequest.w, this.saveArgs.memoryRequest.c);
+								let ml = this.schedulingParamsToEdit(this.saveArgs.memoryLimit.w, this.saveArgs.memoryLimit.c);
```

Comment:
Might be better to have this just take in the whole object as a single parameter and then have it pull out the worker/coordinator values. Less chance of errors from using the wrong property in the wrong place.